lib/ui/views/intro/intro_welcome.dart

```diff
@@ -258,3 +81,328 @@ class _IntroWelcomeState extends ConsumerState<IntroWelcome> {
+                              localizations.welcomeDisclaimerLink,
+                            );
+                          },
+                        ),
+                      ),
+                    ),
+                  ),
+                ],
+              ),
+            Padding(
+              padding: const EdgeInsets.only(
+                right: 30,
+              ),
+              child: Row(
+                mainAxisAlignment: MainAxisAlignment.end,
+                children: <Widget>[
+                  Consumer(
+                    builder: (context, ref, child) {
+                      final asyncVersionString = ref.watch(
+                        versionStringProvider(
+                          AppLocalization.of(context)!,
+                        ),
+                      );
+
+                      return Text(
+                        asyncVersionString.asData?.value ?? '',
+                        textAlign: TextAlign.left,
+                        style: theme.textStyleSize10W100Primary,
+                      );
+                    },
+                  ),
+                ],
+              ),
+            ),
+          ],
+        ),
+      ],
+    );
+  }
+}
+
+class _ButtonNewWallet extends ConsumerWidget {
+  const _ButtonNewWallet({required this.checkedValue});
+
+  final bool checkedValue;
+
+  @override
+  Widget build(BuildContext context, WidgetRef ref) {
+    final localizations = AppLocalization.of(context)!;
+
+    return _ButtonAction(
+      key: const Key('newWallet'),
+      outline: !checkedValue,
+      label: localizations.newWallet,
+      onPressed: () async {
+        if (checkedValue) {
+          await ref.read(SettingsProviders.settings.notifier).setNetwork(
+                const NetworksSetting(
+                  network: AvailableNetworks.archethicMainNet,
+                  networkDevEndpoint: '',
+                ),
+              );
+          Navigator.of(context).pushNamed(
+            '/intro_welcome_get_first_infos',
+          );
+        }
+      },
+    );
+  }
+}
+
+class _ButtonImportWallet extends ConsumerWidget {
+  const _ButtonImportWallet({required this.checkedValue});
+
+  final bool checkedValue;
+
+  @override
+  Widget build(BuildContext context, WidgetRef ref) {
+    final localizations = AppLocalization.of(context)!;
+
+    return _ButtonAction(
+      key: const Key('importWallet'),
+      outline: !checkedValue,
+      label: localizations.importWallet,
+      onPressed: () async {
+        if (checkedValue) {
+          await NetworkDialog.getDialog(
+            context,
+            ref,
+            ref.read(
+              SettingsProviders.settings.select(
+                (settings) => settings.network,
+              ),
+            ),
+          );
+          Navigator.of(context).pushNamed('/intro_import');
+        }
+      },
+    );
+  }
+}
+
+class _ButtonAction extends ConsumerWidget {
```

Comment:
Could you change the name of ``outline``param with ``enabled`` please
And the button import should have ``Dimens.buttonBottomDimens`` and create button ``  Dimens.buttonTopDimens``
I prefer you let buttons like before (perhaps in this case, you are going too far in structuration ;) )